dao/accumulo.rya/src/test/java/org/apache/rya/accumulo/MiniAccumuloClusterInstance.java

```diff
@@ -76,6 +80,34 @@ public void stopMiniAccumulo() throws IOException, InterruptedException {
     }
 
     /**
+     * Copies the HADOOP_HOME bin directory to the {@link MiniAccumuloCluster} temp directory.
+     * {@link MiniAccumuloCluster} expects to find bin/winutils.exe in the MAC temp
+     * directory instead of HADOOP_HOME for some reason.
+     * @throws IOException
+     */
+    private static void copyHadoopHomeToTemp() throws IOException {
```

Comment:
So, this isn't the only class that spins up a Mini Accumulo. I think all of the Fluo tests are using a recipe that Fluo puts out, and that also uses Mini Accumulo. Do all of those tests pass on Windows also?